arm/compute/client.go

```diff
@@ -1,3 +1,7 @@
+// Package compute implements the Azure ARM Compute service API version
+// 2015-06-15.
+//
+// The Compute Management Client..
```

Comment:
this doesn't look very good. :( maybe omit what services describe in swagger and just keep the first line you have? It just seems sufficient to me.